modules/rostests/rosautotest/CConfiguration.cpp

```diff
@@ -75,6 +78,12 @@ CConfiguration::ParseParameters(int argc, wchar_t* argv[])
                     m_Submit = true;
                     break;
 
+                case 't':
+                    ++i;
+                    tmp_RepeatCount = _wtoi(argv[i]);
```

Comment:
Crashes here when running "rosautotest /t" (without any number)

Reply:
Yes, because of "++i" then argv[i], with an incorrect assumption on the array size.

Reply:
The same is probably true of the comment case above then?